test/verify/check-docker-storage

```diff
@@ -23,63 +23,27 @@ import testvm
 
 from testlib import *
 
-# The Docker Storage Setup behaves differently depending on
-# whether the "atomic" utility is recent enough, on whether or not
-# the OS disk is part of a volume group, and on the storage driver
-# that Docker uses by default.
-#
-# These predicates here encode what we expect right now.  If
-# conditions on the distributions change, they need to be updated, but
-# the rest of the test should be able to stay the same.
+non_manageable_images = [
+    # No "atomic" utility
+    "debian-stable",
+    "debian-testing",
+    "ubuntu-1604",
+    "ubuntu-stable"
+]
+
+rootfs_not_in_volume_group_images = [
+    "centos-7",
+    "rhel-7",
+    "rhel-7-4"
```

Comment:
Note, this will collide with #8984. If that lands first, the two rhel*s will (most probably) be dropped, unless you forgot rhel-7-5 here.

Reply:
This list reflects how we build our images, and is not about how the OS inherently behaves.  When using virt-builder, we get no LVM, but when using virt-install, we do get LVM for the root.

(That's one reason why I think we should keep using virt-install even for those images that are available for virt-builder.)

So if we don't change how we build the images, this list will likely not change.